src/data/finset/lattice.lean

```diff
@@ -1108,10 +1122,6 @@ begin
   simp_rw [val_to_finset],
 end
 
-lemma sup_eq_bUnion {α β} [decidable_eq β] (s : finset α) (t : α → finset β) :
-  s.sup t = s.bUnion t :=
-by { ext, rw [mem_sup, mem_bUnion], }
-
```

Comment:
Why did you remove this one?

Reply:
my confusion… Iet me put it back, and write the corresponding lemma for inf

Reply:
Actually, this is not clear how to add an inf, unless one adds a lot of stuff.
(For example, there's no `finset.mem_inf`.)